### PR TITLE
[Docs] Chinese Document of faq

### PR DESCRIPTION
**Pull request by wHao-Wu**
* Add Chinese Document of faq
* Refine a typo in faq

### CODE REVIEW

File: docs_zh-CN/faq.md [hunk 1]
```diff
@@ -1 +1,26 @@
 # 常见问题解答
+
+我们列出了一些用户和开发者在开发过程中会遇到的一些常见问题以及对应的解决方案，如果您发现了任何频繁出现的问题，请随时扩充本列表，并且非常欢迎您提出的任何解决方案。如果您在环境配置、模型训练等工作中遇到任何的问题，请使用[问题模板](https://github.com/open-mmlab/mmdetection3d/blob/master/.github/ISSUE_TEMPLATE/error-report.md/)来创建相应的 issue ,并将所需的所有信息填入到问题模板中，我们会尽快解决您的问题。
```
Comment (THU17cyz) [inline docs_zh-CN/faq.md:3]: 两个“一些”去掉后面一个？

Comment (THU17cyz) [reply docs_zh-CN/faq.md:3]: 并且非常欢迎...去掉“并且”？

Comment (THU17cyz) [reply docs_zh-CN/faq.md:3]: issue后面的逗号是英文的，并且多了一个空格（我感觉英文和标点符号之间应该不用空格？）

Comment (wHao-Wu) [reply docs_zh-CN/faq.md:3]: Solved.

File: docs_zh-CN/faq.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+  ``OSError: /lib/x86_64-linux-gnu/libm.so.6: version 'GLIBC_2.27' not found``
+
+  请将 open3d 的版本降级至 0.9.0.0，因为最新版 open3d 需要 'GLIBC_2.27' 文件的支持， Ubuntu 16.04 系统中缺失该文件，且该文件仅存在于 Ubuntu 18.04 及之后的系统中。
+
+- 如果您遇到下面的问题，并且您的环境包含 numba == 0.48.0 和 numpy >= 1.20.0：
+
+  ``TypeError: expected dtype object, got 'numpy.dtype[bool_]'``
+
+    请将 numpy 的版本降级至 < 1.20.0，或者从源码安装 numba == 0.48，这是由于 numpy == 1.20.0 改变了 API，使得在调用 `np.dtype` 会产生子类。
```
[the remark this answers is not in the document]
Comment (xiliu8006) [reply docs_zh-CN/faq.md:26]: I think we can change the '使得在调用 `np.dtype` 会产生子类' to "more details on this [issue](https://github.com/numba/numba/issues/6041)"， Change it on English version please

**Review by xiliu8006 — APPROVED** [review]

**Comment by wHao-Wu** [comment]
@Tai-Wang 
Comments resolved.

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**